junit5/src/main/java/org/jboss/weld/junit5/auto/ClassScanning.java

```diff
@@ -255,14 +253,16 @@ private static Extension createExtension(Class<? extends Extension> clazz) {
         }
     }
 
+    private static boolean isBeanParameterAnnotation(Annotation ann) {
+        return isAnnotated(ann.annotationType(), Qualifier.class) ||
+                isAnnotated(ann.annotationType(), Scope.class) ||
+                isAnnotated(ann.annotationType(), NormalScope.class);
+    }
+
     private static boolean hasBeanDefiningAnnotation(Class<?> clazz) {
-        return
-                AnnotationUtils.isAnnotated(clazz, ApplicationScoped.class) ||
-                        AnnotationUtils.isAnnotated(clazz, SessionScoped.class) ||
-                        AnnotationUtils.isAnnotated(clazz, ConversationScoped.class) ||
-                        AnnotationUtils.isAnnotated(clazz, RequestScoped.class) ||
-                        AnnotationUtils.isAnnotated(clazz, Dependent.class) ||
-                        AnnotationUtils.isAnnotated(clazz, Stereotype.class);
+        return isAnnotated(clazz, NormalScope.class) || isAnnotated(clazz, Scope.class) ||
```

Comment:
`isAnnotated(clazz, Scope.class)` needs to check for just `Dependent` because no other non-normal scope annotation is considered bean defining according to specification.